src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div class="container"
+      data-qa = "card">
+      <img src="./images/imac.jpeg"
+        alt="imac"
+        class="container__image">
```

Comment:
```suggestion
      <img 
        src="./images/imac.jpeg"
        alt="imac"
        class="container__image"
      >
```
Форматируй теги таким образом